src/Components/Slider/Slide.js

```diff
@@ -3,25 +3,32 @@ import React from 'react';
 import { SlideItem, InfoContainer, Header, Description } from './styled';
 
 const Slide = ({ width, ...props }) => {
-    const {
-        title,
-        description:[
-            {text: productDescription}
-        ],
-        main_image:{
-            url: imageURL
-        }
-    } = props.data;
-
-    return (
-        <SlideItem width={width} content={imageURL}>
-            <InfoContainer>
-                <Header>{title.toLowerCase()}</Header>
-                <Description>{productDescription.length > 30 ? productDescription.slice(0,30): productDescription}</Description>
-            </InfoContainer>
-            
-        </SlideItem>
-    );
+
+
+    if(props.data){
```

Comment:
If you need `data` and `image`, destructure the props to extract them in the arguments. Rather than sending everything to props with `...props`.